include/libXbSymbolDatabase.h

```diff
@@ -197,15 +230,18 @@ void XbSymbolDatabase_SetOutputMessage(xb_output_message_t message_func);
 
 /// <summary>
 /// To register any detected symbol name with address and build version back to third-party program.
-/// NOTE: Be aware of library name will be varity since some libraries are detecting in other sections as well.
+/// NOTE: Be aware of library name will be variety since some libraries are detecting in other sections as well.
```

Comment:
```suggestion
/// NOTE: Be aware that the library name will vary since some libraries are detected in other sections as well.
```

Reply:
This was a typo since Visual Studio's spell checking marked as misspelled.